src/algebra.ts

```diff
@@ -25,16 +25,14 @@ export interface Scalar
   /// scalar is not in the right range. THis method
   /// is to be used to create arrays from any streams
   /// of bytes (think transcript hashing).
-  fromBytes(array: Uint8Array): this;
+
+  // TODO: Should we remove this?
+  // fromBytes(array: Uint8Array): this;
```

Comment:
yes

Reply:
Ok great. There's also a method `setHashOf` that we don't use because it's not implemented in ffjavascript. Can I remove that as well?